Ginger/GingerCoreNET/ActionsLib/ActionConversionUtils.cs

```diff
@@ -0,0 +1,266 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using GingerCore;
+using GingerCore.Actions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    /// <summary>
+    /// This class is used to add methods for action conversion helpers
+    /// </summary>
+    public class ActionConversionUtils
+    {
```

Comment:
why class name was not changed to "ConvertableActionDetails"?

Reply:
Initially we decided to name something like "utils" so kept such name, now changed the name to "ConvertableActionDetails"